src/main/scala/tv/teads/tour/basics/FinalExam.scala

```diff
@@ -0,0 +1,13 @@
+package tv.teads.tour.basics
```

Comment:
We should have another sub package named stepX to make sure the files are read in the order we want them to be